autogpt_platform/backend/backend/util/logging.py

```diff
@@ -1,12 +1,17 @@
-import os
+from backend.util.settings import AppEnvironment, BehaveAs, Settings
+
+settings = Settings()
 
 
 def configure_logging():
     import logging
 
     import autogpt_libs.logging.config
 
-    if os.getenv("APP_ENV") != "cloud":
+    if (
+        settings.config.behave_as == BehaveAs.LOCAL
+        or settings.config.app_env == AppEnvironment.LOCAL
+    ):
```

Comment:
This was previously impossible as cloud isn't a value field in the AppEnvironment Enum so the application would never start with this set to cloud. 

This would have prevented cloud logging from ever being tested so do be careful with this. We may want to disable this for now. @aarushik93